version/version.go

```diff
@@ -9,7 +9,7 @@ import (
 
 var (
 	// Version is the main version number that is being run at the moment.
-	Version = "2.0.0"
+	Version = "1.4.3"
```

Comment:
Why do we make it v1.4.3? I thought this fork was for v2

Reply:
I only update the version when I"m releasing and just did this to resolve merge conflicts against the main branch quicker, I'll change this back to v2 when we release on main